3-mcstas/make_powder_instrument.py

```diff
@@ -0,0 +1,7 @@
+import mcstasscript as ms
+
+
+def make(**kwargs):
```

Comment:
My idea was actually to just exclude the notebooks from execution instead of making dummy functions...

Reply:
I actually just did that because there were still some failures.